__tests__/app.test.js

```diff
@@ -158,5 +158,135 @@ describe('app.js tests', () => {
           expect(body.msg).toBe('Endpoint not found');
         });
     });
-  });  
+  });
+
+  describe('PATCH /api/reviews/:review_id', () => {
+    // functionality
+    test('it accepts object with key inc_votes and value of number (can be both positive and negative) and returns object with key of updated containing updated object', () => {
+      return request(app)
+        .patch('/api/reviews/1')
+        .send({
+          inc_votes: 1,
+        })
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toHaveProperty('updated');
+          expect(Array.isArray(body)).toBe(false);
+          expect(Array.isArray(body.updated)).toBe(false);
+          expect(typeof body).toBe('object');
+          expect(typeof body.updated).toBe('object');
+        });
+    });
+    test('object with positive value increases', () => {
+      return request(app)
+        .patch('/api/reviews/1')
+        .send({
+          inc_votes: 10,
+        })
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.updated.votes).toBe(11);
+        });
+    });
+    test('object with negative value decreases', () => {
+      return request(app)
+        .patch('/api/reviews/1')
+        .send({
+          inc_votes: -1,
+        })
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.updated.votes).toBe(0);
+        });
+    });
+    test('object with negative value cant go below 0', () => {
+      return request(app)
+        .patch('/api/reviews/1')
+        .send({
```

Comment:
You don't need this logic. There could be reviews with votes in the minuses if they are poor reviews! 😅